### PR TITLE
Adding multi node xgboost dask

### PR DESCRIPTION
**Pull request by arunraman**
Adding xgboost Multi-node dask example for GCP AI Platform.

### CODE REVIEW

File: ai-platform-samples/xgboost_multi_node/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# We modify the original Higgs data into 1GB chunks and append to different size for benchmarking. We also generate parquet files for our testing.
+
+FROM mengdong/test-rapids:latest
```
Comment (mengdong) [inline ai-platform-samples/xgboost_multi_node/Dockerfile:18]: could you replace the image to rapids ngc image?

File: ai-platform-samples/xgboost_multi_node/README.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+|---|---|---|---|---|---|
+|  1 | 1 | 2 | 2 + 1 | 4 | 228.21  |
+|  2 | 2 | 4 | 4 + 1 | 4 |  177.80 |
+|  3 | 2 | 4 | 8 + 1 | 4 |  155.40 |
+
+
+|  Experiment | No of GPUs/ Worker  |  num-workers  | Total No of GPU used | nthreads  | Xgb.dask.train time (sec) |
+|---|---|---|---|---|---|
+|  1 | 1 | 2 | 2 + 1 | 8 | 268.25  |
+|  2 | 2 | 4 | 4 + 1 | 8 |  153.21 |
```
Comment (mengdong) [inline ai-platform-samples/xgboost_multi_node/README.md:56]: what is the math for total no of GPU here? isn't 2*4=8?

File: ai-platform-samples/xgboost_multi_node/README.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+## Results
+
+### Parquet
+
+Dataset :  gs://<gcs_bucket>/rapids-higgs-parquet/*.parquet (10 GiB)
+
+|  Experiment | No of GPUs/ Worker  |  num-workers  | Total No of GPU used | nthreads  | Xgb.dask.train time (sec) |
+|---|---|---|---|---|---|
+|  1 | 1 | 2 | 2 + 1 | 4 | Memory Error  |
+|  2 | 2 | 4 | 4 + 1 | 4 |  138.06 |
```
Comment (mengdong) [inline ai-platform-samples/xgboost_multi_node/README.md:31]: why 2 gpu per worker?

File: ai-platform-samples/xgboost_multi_node/rapids_distributed.yaml [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+  masterConfig:
+    acceleratorConfig:
+      count: '1'
+      type: NVIDIA_TESLA_T4
+    imageUri: gcr.io/<account_name>/gcp_rapids_dask:latest>
+  masterType: n1-highmem-4
+  scaleTier: CUSTOM
+  workerConfig:
+    acceleratorConfig:
+      count: '2'
```
Comment (mengdong) [inline ai-platform-samples/xgboost_multi_node/rapids_distributed.yaml:35]: why 2 gpu per worker?

File: ai-platform-samples/xgboost_multi_node/README.md (hunk 3 above)
Comment (arunraman) [reply ai-platform-samples/xgboost_multi_node/README.md:31]: This table explains various scenarios of GPU/worker combination.

File: ai-platform-samples/xgboost_multi_node/README.md (hunk 2 above)
Comment (arunraman) [reply ai-platform-samples/xgboost_multi_node/README.md:56]: Num of GPUs  = Num of GPU in master + Number of GPUs/worker * worker

File: ai-platform-samples/xgboost_multi_node/Dockerfile (hunk 1 above)
Comment (arunraman) [reply ai-platform-samples/xgboost_multi_node/Dockerfile:18]: sure

**Comment by mengdong** [comment]
@roarjn @ethem-kinginthenorth  could you please review and let Arun know if any additional change needed?

**Comment by arunraman** [comment]
Currently removing the results from the README

**Comment by mengdong** [comment]
LGTM

**Pull request merged**